tools/SetupFlow/DevHome.SetupFlow/Views/AddRepoDialog.xaml

```diff
@@ -152,7 +152,8 @@
                         VerticalAlignment="Center"
                         Command="{x:Bind AddRepoViewModel.CancelButtonPressedCommand, Mode=OneWay}"
                         DataContext="{x:Bind}"
-                        Style="{ThemeResource AlternateCloseButtonStyle}">
+                        Style="{ThemeResource AlternateCloseButtonStyle}"
+                        AutomationProperties.Name="Close (X)">

```

Comment:
If this gets read it has to be localized. Example: https://github.com/microsoft/devhome/blob/a0f089262a8b4239caa607671a934524b5ceb24f/common/Strings/en-us/Resources.resw#L124